script/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+SIMPLEFS_MOD=simplefs.ko
+IMAGE=$1
+IMAGESIZE=$2
+MKFS=$3
+
+test_op() {
+    local op=$1 
+    echo -n "Testing cmd: $op..."
+    sudo $op >/dev/null && echo "Success"
+}
+
+check_exist() {
+    local mode=$1
+    local nlink=$2
+    local name=$3
+    echo -n "Check if exist: $mode $nlink $name..."
+    sudo ls -lR  | grep -e "$mode $nlink".*$name >/dev/null && echo "Success" || \
+    echo "Failed" 
+}
+
+if [ "$EUID" -eq 0 ]
+  then echo "Don't run this script as root"
+  exit
+fi
+
+mkdir -p test  
+sudo umount test 2>/dev/null && \
+sudo rmmod $SIMPLEFS_MOD 2>/dev/null
+sleep 1
+modinfo $SIMPLEFS_MOD || exit 1
+sudo insmod $SIMPLEFS_MOD && \
+dd if=/dev/zero of=$IMAGE bs=1M count=$IMAGESIZE && \
+./$MKFS $IMAGE && \
+sudo mount -t simplefs -o loop $IMAGE test
```

Comment:
Here is `mount`. But when would you execute `umount`?

Reply:
I originally wanted to keep it there for manual testing, but it doesn't seem to make sense though.